lib/convection/model/exceptions.rb

```diff
@@ -0,0 +1,17 @@
+ExcessiveResourcesError = Class.new( StandardError)
+ExcessiveResourceNameError = Class.new( StandardError)
+ExcessiveMappingsError = Class.new( StandardError)
+ExcessiveMappingAttributesError = Class.new( StandardError)
+ExcessiveMappingNameError = Class.new( StandardError)
+ExcessiveMappingAttributeNameError = Class.new( StandardError)
+ExcessiveParametersError = Class.new( StandardError)
+ExcessiveParameterNameError = Class.new( StandardError)
+ExcessiveParameterBytesizeError = Class.new( StandardError)
+ExcessiveOutputsError = Class.new( StandardError)
+ExcessiveOutputNameError = Class.new( StandardError)
+ExcessiveDescriptionError = Class.new( StandardError)
+ExcessiveTemplateSizeError = Class.new( StandardError)
+
+def LimitExceededError(value, limit, errorClass)
+  raise errorClass, "Value #{value} exceeds Limit #{limit}"
+end
```

Comment:
Method names should be under_score_and_lower_case.  It looks like this method is a class being called wrong in the code.

https://github.com/rapid7/ruby-style-guide/blob/master/README.md#naming